src/util/CurrencyInfoHelpers.ts

```diff
@@ -82,6 +82,26 @@ export const getTokenIdForced = (account: EdgeAccount, pluginId: string, currenc
   return tokenId
 }
 
+export function getCurrencyCodeMultiplier(currencyConfig: EdgeCurrencyConfig, currencyCode: string): string {
+  const { currencyInfo, allTokens } = currencyConfig
+  for (const denomination of currencyInfo.denominations) {
+    if (denomination.name === currencyCode) {
+      return denomination.multiplier
+    }
+  }
+
+  for (const tokenId of Object.keys(allTokens)) {
+    const token = allTokens[tokenId]
```

Comment:
You could just use `Object.values` and avoid the `tokenId` allocation altogether.